app/code/Magento/Email/Model/Template/Filter.php

```diff
@@ -1086,7 +1086,7 @@ public function filter($value)
             if ($this->_appState->getMode() == \Magento\Framework\App\State::MODE_DEVELOPER) {
                 $value = sprintf(__('Error filtering template: %s'), $e->getMessage());
             } else {
-                $value = __("We're sorry, an error has occurred while generating this content.");
+                $value = (string) __("We're sorry, an error has occurred while generating this content.");
```

Comment:
Hello, @Skullsneeze! Your fix looks reasonable, but our workflow requires obvious steps to reproduce a fixed issue. I was unable to find any scenario for reproducing your issue. Please, add a testing scenario to this PR and we should process with this PR.